crates/iota-types/src/messages_grpc.rs

```diff
@@ -272,3 +273,12 @@ pub struct HandleSoftBundleCertificatesRequestV1 {
     pub include_output_objects: bool,
     pub include_auxiliary_data: bool,
 }
+
+#[derive(Clone, Debug, Serialize, Deserialize)]
+pub struct HandleCapabilityNotificationV1Request {
+    pub message: SignedAuthorityCapabilitiesV1,
+}
+#[derive(Clone, Debug, Serialize, Deserialize)]
+pub struct HandleCapabilityNotificationV1Response {
+    // Currently empty, may be extended in the future
```

Comment:
Breaking changes are fine here or how would this be extended in the future?

Reply:
Good point, I guess changing this in the future would be problematic. I'll remove the comment and I'll double check if something needs to be returned here.